crates/chia-consensus/src/gen/conditions.rs

```diff
@@ -1330,7 +1329,7 @@ pub fn validate_conditions(
     ret: &SpendBundleConditions,
     state: ParseState,
     spends: NodePtr,
-    flags: u32,
+    _flags: u32,
```

Comment:
Does this need to be here still?

Reply:
it doesn't. I started removing it but left it in as I think it's quite nice to have the infrastructure to affect conditions parsing by new flags